docs/developing.md

```diff
@@ -189,44 +185,26 @@ To remove all generated artifacts and installed dependencies, run:
 <a name="releasing-minishift"></a>
 ## Releasing Minishift
 
-**Note**: The following paragraphs are partly out of date. Work in progress ...
+<a name="prerequisites-1"></a>
+### Prerequisites
 
-<a name="create-a-release-notes-pr"></a>
-### Create a Release Notes PR
+* A [GitHub personal access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use)
+  defined in your environment as GITHUB_ACCESS_TOKEN
 
-Assemble all the meaningful changes since the last release into the CHANGELOG.md file.
-See [this PR](https://github.com/kubernetes/minikube/pull/164) for an example.
+<a name="cutting-the-release"></a>
+### Cutting the release
 
-<a name="build-and-release-a-new-iso"></a>
-### Build and Release a New ISO
+* Update the [CHANGELOG.md](../CHANGELOG.md) with the changes since the last release.
+* Bump the version in the [VERSION](../VERSION)
+* Commit
+* Tag the release locally. Something like:
 
-This step isn't always required. Check if there were changes in the deploy directory.
-If you do this, bump the ISO URL to point to the new ISO, and send a PR.
+        $ git tag -a v1.0.0 -m "1.0.0 Release"
```

Comment:
he he - touché. Good point. I wrote this before I understood myself what exactly gh-release does. This is in fact obsolete. Let me clean this up.